network-api/networkapi/wagtailpages/templates/wagtailpages/publication_page.html

```diff
@@ -48,35 +48,25 @@ <h1 class="h1-heading">
         </div>
         {{ page.contents_title }}
 
-    {% if page.is_chapter_page %}
-      {# Chapter index page; display child article pages #}
-      <div class="container-sm background-white chapter-body">
-        {% for child in page.get_child_article_pages %}
-          <div class="row d-flex align-items-center">
-            <div class="col">
-              <div class="chapter-number">
-                <div>
-                  {{ page.get_chapter_number_or_none }}.{{ forloop.counter }}
-                </div>
-              </div>
-            </div>
-            <div class="col-md-11">
-              <a href="{{ child.url }}" class="d-block w-100 h3 chapter-article-link">
-                {{ child.title }}
-              </a>
-            </div>
-            {% if not forloop.last %}
-              <div class="col-12 my-3">
-                <hr>
-              </div>
+      {# page summary #}
+      <div class="child-pages-summary">
+      {% comment %} https://django-treebeard.readthedocs.io/en/stable/api.html#treebeard.models.Node.get_annotated_list {% endcomment %}
+        {% for child, node_info in page.my_children %}
+            {% comment %} 'open' means 'there are siblings so open the list tag' {% endcomment %}
+            {% if node_info.open %}
+                <ol><li data-chapter="{{page.get_chapter_number_or_none|default_if_none:''}}">
+            {% else %}
+                </li><li data-chapter="{{page.get_chapter_number_or_none|default_if_none:''}}">
```

Comment:
What if there are chapter pages? This is assuming all child pages will be Article Pages, is it not? 

An example would be the design below, what is there's:
1. Article Page 
2. Chapter Page 
3. Article Page 
4. Chapter Page 
5. etc etc. 
![image](https://user-images.githubusercontent.com/4743971/91453352-d7894200-e83c-11ea-874c-0636529a3ba9.png)

Reply:
It will list any type of child page. Conceptually, in that case the ArticlePage is still a chapter, even if its a single page chapter.

Reply:
![image](https://user-images.githubusercontent.com/4743971/91454684-81b59980-e83e-11ea-88dd-9b6b31986b90.png)

This is a Publication Page -> Publication Page -> Publication Page, it's mixed in with the Article Pages. 

I'm thinking we should create the Chapter Page at this point. Using a Publication Page as 2 different page types is getting too complex. We could create a PublicationPage and use `class ChapterPage(PublicationPage)` and add overwrites. But the newly specified content_type could make traversing easier. 

Let's have a chat about this before continuing.